projects/tinkerbell/charts/docker/linux/relay-init/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nsenter
```

Comment:
what is relay-init used for?

Reply:
Same with this one, i'll remove from the pr. Relay init will be used for https://github.com/tinkerbell/charts/blob/43a6c2ce1056f1d409ddef8ee7d45c33138e763c/tinkerbell/stack/templates/nginx.yaml#L157